src/com/hillel/Main.java

```diff
@@ -1,8 +1,7 @@
 package com.hillel;
 
 public class Main {
-
     public static void main(String[] args) {
-        System.out.println("Hello Java");
+        System.out.println("Hello Java Pro!!!");
```

Comment:
Please, make your changes in your separate package (name it with your name)

Reply:
The separate package was created.